ddtrace/contrib/pytest/_plugin_v2.py

```diff
@@ -56,13 +58,18 @@ def _handle_itr_should_skip(item, test_id) -> bool:
     if not CISession.is_test_skipping_enabled():
         return False
 
-    item_is_unskippable = CITest.is_item_itr_unskippable(item) or CISuite.is_item_itr_unskippable(test_id.parent_id)
+    suite_id = test_id.parent_id
 
-    if CISuite.is_item_itr_skippable(test_id.parent_id):
+    item_is_unskippable = CISuite.is_item_itr_unskippable(suite_id)
+
+    if CISuite.is_item_itr_skippable(suite_id):
         if item_is_unskippable:
```

Comment:
@romainkomorndatadog  As I understand, this can only be true if the suite passes both `is_item_itr_skippable` and `is_item_itr_unskippable`. Can this happen? Why do we have both functions?